test/benchmark-queue-work.c

```diff
@@ -0,0 +1,47 @@
+#include "task.h"
+#include "uv.h"
+
+static int done = 0;
+static unsigned events = 0;
+
+static unsigned fastrand(void) {
+  static unsigned g = 0;
+  g = g * 214013 + 2531011;
+  return g;
+}
+
+static void work_cb(uv_work_t *req) { 
+  fastrand();
+}
+
+static void after_work_cb(uv_work_t *req, int status) {
```

Comment:
```suggestion
static void after_work_cb(uv_work_t* req, int status) {
```
Ditto on lines 25 and 30.

Reply:
@bnoordhuis, you don't happen to have a clang format for libuv?